src/systemdhook.c

```diff
@@ -819,12 +853,75 @@ int main(int argc, char *argv[])
 	char *mount_label = NULL;
 	const char **config_mounts = NULL;
 	unsigned config_mounts_len = 0;
+	unsigned array_len = 0;
 
 	/* Extract values from the config json */
 	const char *mount_label_path[] = { "linux", "mountLabel", (const char *)0 };
 	yajl_val v_mount = yajl_tree_get(config_node, mount_label_path, yajl_t_string);
 	mount_label = v_mount ? YAJL_GET_STRING(v_mount) : "";
 
+	/* Get the gid value. */
+	int gid = -1;
+	const char *gid_mappings[] = {"linux", "gidMappings", (const char *)0 };
+	yajl_val v_gidMappings = yajl_tree_get(config_node, gid_mappings, yajl_t_array);
+	if (!v_gidMappings) {
+		pr_pinfo("gidMappings not found in config");
+		gid=0;
+	}
+
+	const char *container_path[] = {"containerID", (const char *)0 };
+	if (gid != 0) {
```

Comment:
Ok if I leave this as is and change in a follow up?  Ditto below comment.  This is working as is, I'd like to keep it at least for tonight to get it in.  Personally I prefer it as is, but can change it if you think it's better.